src/components/pages/VolunteersPage/VolunteersPage.js

```diff
@@ -124,7 +124,10 @@ class VolunteersPage extends React.Component {
               <>
```

Comment:
Wrong PR but would prefer `<div key={volunteerGroup.letter}> ... </div>` here instead of fragment (`<> ... </>`).